README.md

```diff
@@ -51,3 +51,12 @@ While we will be unable to enforce social distancing guidelines during the UGM,
 Written materials are provided under the [CC-BY-4.0 SA license](LICENSE.md).
 
 The MDAnalysis logo and its derivatives are licensed under the Creative Commons Attribution-NoDerivs 3.0 Unported License.
+
+## Acknowledgements
+
+Please see [`AUTHORS.md`](AUTHORS.md) for a list of contributors to the workshop
+materials.
+
+The MDAnalysis UGM has been made possible in part by CZI grants; DAF2021-237663 (grant DOI https://doi.org/10.37921/426590wiobus), 2022-253062 from the Chan Zuckerberg Initiative DAF, an advised fund of Silicon Valley Community Foundation (funder DOI 10.13039/100014989).
```

Comment:
@IAlibay I'm not sure why you've put the first CZI grant?  I've added the grant number for the second, it doesn't have a DOI

Reply:
Because everyone working on the MDAKits hackathon is paid under EOSS4?

Reply:
I can remove it if you think it's not necessary though.

Reply:
I would list EOSS5 first but as Irfan said, EOSS4 is paying for MDAKits stuff and as you remember, we decided to roll the final EOSS4 meeting into the UGM.